server/controllers/users/Update.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../../models/User')
+const errors = require('../../../shared/errors')
+
+module.exports = async function createUser(ctx) {
+  const { id } = ctx.params
+  const payload = ctx.request.body
+
+  const validRequest = [payload.password, id].every(item => item !== undefined)
+  if (!validRequest) {
+    ctx.status = 400
+    ctx.body = { code: errors.INVALID_REQUEST }
+    return
+  }
+
+  const user = await User.where({ id }).fetch()
+  if (user === null) {
+    ctx.status = 404
+    ctx.body = { code: errors.NOT_FOUND }
+    return
+  }
+
+  await user.save(payload)
```

Comment:
Como você me garante que não tem nada a mais que deveria aqui?

Faltou usar sua função de validação...

Reply:
Bem, quanto a função de validação.
Está numa branch diferente, então só vou aplicar a está rota quando ela for pro master.
Quis deixar essa independente pra poder ir logo sem ter que esperar a outra q é mais demorada.

De todo modo, eu deixei um lembrete na pr de validações pra lembrar de rever isso quando for dado o merge.